benchmarks/src/garage_benchmarks/experiments/algos/ppo_garage_tf.py

```diff
@@ -68,7 +68,7 @@ def ppo_garage_tf(ctxt, env_id, seed):
                       optimizer_args=dict(
                           batch_size=32,
                           max_epochs=10,
-                          tf_optimizer_args=dict(learning_rate=3e-4),
+                          optimizer_args=dict(learning_rate=3e-4),
```

Comment:
hmm this is confusing -- optimizer_args has an optimizer_args inside of it?

perhaps we should just make these inner `optimizer_args` part of the outer `optimizer_args` and then just pass them directly.

Reply:
Cleaned out nested `optimizer_args `